lambdas/death-enrichment/src/main/java/uk/gov/di/data/lep/GroDeathEnrichment.java

```diff
@@ -42,15 +42,15 @@ public GroDeathEventEnrichedData handleRequest(SQSEvent sqsEvent, Context contex
 
     @Tracing
     private GroDeathEventEnrichedData enrichData(GroJsonRecord baseData) {
-        logger.info("Enriching and mapping data (sourceId: {})", baseData.registrationId());
+        logger.info("Enriching and mapping data (sourceId: {})", baseData.registrationID());
 
         return new GroDeathEventEnrichedData(
-            baseData.registrationId(),
+            baseData.registrationID(),
```

Comment:
Presumably we actually want (when this change is all done) to use the new types here instead of this one, anyway?

Reply:
Yes, just got frustrated and did it here too

Reply:
👍